style.css

```diff
@@ -0,0 +1,856 @@
+*{
+	box-sizing: border-box;
+}
+
+.container.fluid {
+    display: block;
+    background: rgb(0, 255, 242);
+    border: 1px solid brown;
+}
+
+.header-top {
+    background-color: #f72210;
+    color: white;
+    /*height: 95px;*/
+    /*padding: 20px 10px 15px;*/
+    position: relative;
+}
+
+.header-top a {
+    color: white;
+    font-size: 16px;
+}
+
+.middle img {
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    width: 55%; 
+}
+
+.page-header .date {
+    display: block;
+    margin-left: 10px;
+    margin-top: 35px;
+}
+
+.header-top .left-side {
+    font: bold 14px/14px "Roboto Condensed",sans-serif;
+    text-transform: uppercase;
+    justify-content: flex-end;
+    align-items: flex-end;
+}
+
+.header-top .left-side a {
+    text-decoration: none;
+    color: white;
+}
+
+.header-top .left-side .button {
+    background-color: white;
+    color: #f72210;
+}
+
+@media screen and (max-width: 768px) {
+.header-top .date { visibility: hidden; margin-top: 1px}
+.middle img {position: absolute; width: 46%; left:0px;}
+.header-top .left-side { position: absolute; right: 0; bottom: 5px;}
+.header-top a {font-size: 11px;}
+}
+
+.header-bottom {
+    padding-right: 160px;
+    background-color: white;
+    border-bottom-color: #f4f5f6;
+    padding-top: 15px;
+    border-bottom-width: 2px;
+    border-bottom-style: solid;
+    position: relative;
+}
+
+.header-bottom li {
+    display: inline-block;
+    font-size: 20px;
+}
+
+ul.page-nav {
+    font-size: 21px;
+    line-height: 24px;
+    font-family: "Roboto Condensed",sans-serif;
+    margin-left: 1%;
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
#### `HINT`

Consider making the media query screens around `990px` instead. This will prevent the large navbar from displaying inappropriately. See below:


| Clone | Source |
| :-----: | :-----: |
| ![newsweek lg res nav clone](https://user-images.githubusercontent.com/36057474/63238415-4d2bfc80-c23e-11e9-8a55-c39b95a6131e.gif)  | ![newsweek lg res nav source](https://user-images.githubusercontent.com/36057474/63238408-456c5800-c23e-11e9-9584-ddde2a8d5978.gif)   |